src/common.js

```diff
@@ -166,14 +166,14 @@ function applyPackages (id, packages) {
     const pkg = packages[pkgName];
     if (pkg === null) return;
     if (id.length > pkgName.length && pkg[pkg.length - 1] !== '/')
-      targetWarning(pkgName, pkg, "should have a trailing '/'");
+      targetWarning(6, pkgName, pkg, DEV && "should have a trailing '/'");
     else
       return pkg + id.slice(pkgName.length);
   }
 }
 
-function targetWarning (match, target, msg) {
-  console.warn("Package target " + msg + ", resolving target '" + target + "' for " + match);
+function targetWarning (code, match, target, msg) {
+  console.warn(errMsg(code, DEV ? "Package target " + msg + ", resolving target '" + target + "' for " + match : [msg, target, match]))
```

Comment:
I'd prefer to not make this `errMsg` function polymorphic taking an object or string - rather can e just always make the message a string like eg `DEV ? "..." : [msg, target, match].join(', ')

Repeating such a call isn't a problem for code size as gzip will turn them all into a single reference.